### PR TITLE
making links distinguishable

### PR DESCRIPTION
**Pull request by ShaguftaMethwani**

### CODE REVIEW

**Comment by pypingou** [comment]
Code wise it looks good, do you have a screenshot of how it looks like ? :)

File: nuancier/static/nuancier.css [hunk 1]
```diff
@@ -5,7 +5,10 @@
 .error, .required {
     color: red;
 }
-
+.hyphen{
+    margin-left:80px;
+    margin-right:80px;
+}
```
Comment (pypingou) [inline nuancier/static/nuancier.css]: Small nitpicking, but I like to keep a blank line between my css declarations, so one before and one after :-)

**Comment by ShaguftaMethwani** [comment]
Attaching the sample screenshot of the code run in an online editor. You get the idea how it will look like.

dd

**Comment by pypingou** [comment]
Ok let's merge, thanks for your work!

**Comment by ShaguftaMethwani** [comment]
:)

**Pull request merged**